app/controllers/catalog_controller.rb

```diff
@@ -300,6 +304,10 @@ def atomic_form_field_changed
         if params[:display]
           page << "miq_tabs_show_hide('#details_tab', '#{(params[:display] == "1")}')"
         end
+        page.replace_html("provision_entry_point", :partial => "provision_entry_points") if params[:fqname_type]
```

Comment:
`fqname_type` is confusing, because `fqname` implies embedded_automate.  Recommend something more general like `entrypoint_type` or `automation_type`